src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -126,6 +138,9 @@ public void writeToPacket(final PacketBuffer buffer) {
         buffer.writeVarLong(this.getStackSize());
         buffer.writeVarLong(this.getCountRequestable());
         buffer.writeItemStack(getDefinition(), false);
```

Comment:
Is it a good idea to rely on writeItemStack here given that we now do the reading ourselves? I'd be for inlining this method here to make it mirror our reading code.

Reply:
Might be an idea. On the other hand we will get an immediate response when something in forge/vanilla changes instead of missing it because we duplicated the current behaviour.